packages/atlas/src/components/ListItem/ListItem.styles.ts

```diff
@@ -52,6 +52,10 @@ export const Container = styled.div<ContainerProps>`
   cursor: pointer;
   background-color: ${({ highlight }) => (highlight ? cVar('colorBackgroundMutedAlpha') : 'unset')};
 
+  :active {
+    text-decoration: none;
+  }
```

Comment:
I still see underline under links(active and not active): 
![image](https://user-images.githubusercontent.com/51168865/178204673-94af8be0-525c-4473-af4c-59c9bfe34c67.png)